### ISSUE TITLE
[gateway-gcs] Support for PROFILE

### ISSUE THREAD

**Issue by krishnasrinivas** [minio/minio#4417]
```
 minio gateway:

  4. Start minio gateway server for GCS backend.
      $ gcloud init --console-only # this will configure your google account
      $ export MINIO_ACCESS_KEY=accesskey
      $ export MINIO_SECRET_KEY=secretkey
      $ minio gateway gcs [PROFILE:]PROJECTID
```

Support for PROFILE is not available as shown in the help here

**Comment by deekoder** [comment]
I believe project id is the only thing needed since its unique. But its better to check on this. Krishna, have you any test where you wanted to use the profile and were unable to?

**Comment by harshavardhana** [comment]
> minio gateway:
> 
>   4. Start minio gateway server for GCS backend.
>       $ gcloud init --console-only # this will configure your google account
>       $ export MINIO_ACCESS_KEY=accesskey
>       $ export MINIO_SECRET_KEY=secretkey
>       $ minio gateway gcs [PROFILE:]PROJECTID
> Support for PROFILE is not available as shown in the help here

There is no easier way to support PROFILE AFAIK, atleast from SDK point of view it doesn't differentiate. Only parameter it takes is `projectID`.

**Comment by deekoder** [comment]
We do have to verify that this is the case.

**Comment by harshavardhana** [comment]
> We do have to verify that this is the case.

@nl5887 already verified this before and said its not possible.

**Comment by krishnasrinivas** [comment]
> @nl5887 already verified this before and said its not possible.

Cool, In that case the help message needs to be fixed.

**Comment by deekoder** [comment]
ok then gr8.

**Comment by harshavardhana** [comment]
Closing this.. as fixed in #4421
